<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtArteryTechPhysicsSetupUI</class>
 <widget class="QDialog" name="QtArteryTechPhysicsSetupUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtArteryTechPhysicsSetupUI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="TotalResistanceLabel">
           <property name="text">
            <string>Total Resistance:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="TotalResistanceLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="TotalCapacityLabel">
           <property name="text">
            <string>Total Capacity:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="TotalCapacityLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="AllImpedanceLabel">
           <property name="text">
            <string>All Impedance:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="AllImpedanceLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="PdForWK3Label">
           <property name="text">
            <string>Pd for WK3:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="PdForWK3LineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="P0ForWK3Label">
           <property name="text">
            <string>P0 for WK3:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="P0ForWK3LineEdit">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="FluidViscosityLabel">
           <property name="text">
            <string>Fluid Viscosity:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="FluidViscosityLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="FluidDensityLabel">
           <property name="text">
            <string>Fluid Density:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="FluidDensityLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="BloodFlowSimulationLabel">
           <property name="text">
            <string>Blood Flow Simulation:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="BloodFlowSimulationLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="FluidOnlySimulationLabel">
           <property name="text">
            <string>Fluid Only Simulation:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="FluidOnlySimulationLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="TimeDiscretizationMethodLabel">
           <property name="text">
            <string>Time Discretization Method:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="TimeDiscretizationMethodLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="InitialTimeLabel">
           <property name="text">
            <string>Initial Time:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="InitialTimeLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="FinalTimeLabel">
           <property name="text">
            <string>Final Time:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="FinalTimeLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="TimeStepSizeLabel">
           <property name="text">
            <string>Time Step Size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="TimeStepSizeLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="PeriodLabel">
           <property name="text">
            <string>Period:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="PeriodLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="FixedMeshLabel">
           <property name="text">
            <string>Fixed Mesh:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="FixedMeshLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QLabel" name="ZeroInitialConditionLabel">
           <property name="text">
            <string>Zero Initial Condition:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ZeroInitialConditionLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <widget class="QLabel" name="InletVelocityLabel">
           <property name="text">
            <string>Inlet Velocity:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="InletVelocityLineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <widget class="QPushButton" name="OkPushButton">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="CancelPushButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
